Build/CMake/Modules/GLTFPlatform.cmake

```diff
@@ -27,22 +27,24 @@ elseif (IOS_PLATFORM)
         set(${outPlatform} iOS PARENT_SCOPE)
     elseif (IOS_PLATFORM STREQUAL "SIMULATOR")
         set(${outPlatform} iOSSimulator PARENT_SCOPE)
-    elseif(IOS_PLATFORM STREQUAL "SIMULATOR64")
+    elseif (IOS_PLATFORM STREQUAL "SIMULATOR64")
         set(${outPlatform} iOSSimulator64 PARENT_SCOPE)
-    elseif (IOS_PLATFORM STREQUAL "TVOS" PARENT_SCOPE)
+    elseif (IOS_PLATFORM STREQUAL "TVOS")
         set(${outPlatform} appleTV PARENT_SCOPE)
     elseif (IOS_PLATFORM STREQUAL "SIMULATOR_TVOS")
         set(${outPlatform} appleTVSimulator PARENT_SCOPE)
     else()
         message(FATAL_ERROR "Invalid IOS_PLATFORM: ${IOS_PLATFORM}")
     endif()
 elseif (MSVC)
-    if (CMAKE_GENERATOR_PLATFORM STREQUAL "Win32")
+    if (CMAKE_GENERATOR_PLATFORM STREQUAL "win32")
```

Comment:
It is worth considering doing a case insensitive comparison just in case we ever build using a different generator platform case.

string(TOLOWER "${CMAKE_GENERATOR_PLATFORM}" CMAKE_GENERATOR_PLATFORM_LOWER)
...
if (CMAKE_GENERATOR_PLATFORM_LOWERSTREQUAL "win32")

Also, this would allow you to revert your changes to wininit.ps1 since the case doesn't matter anymore.

Reply:
My preference is to try to make them consistent everywhere rather than handling inconsistencies, just because I think this will make our lives easier on not-Windows.